components/cartPage/OrderResume.tsx

```diff
@@ -0,0 +1,37 @@
+import { useAppSelector } from "@/lib/hookts";
+import { RootState } from "@/lib/store";
+import React from "react";
+
+export default function OrderResume() {
+  const cartProducts = useAppSelector((state: RootState) => state.cartProducts);
+
+  const totalPrice = cartProducts.reduce(
+    (total, product) => total + product.price * product.quantity,
```

Comment:
@Kamildeeal this could be done inside the selector. Then you just pass it to useAppSelector like
```
const totalPrice = useAppSelector(selectTotalPrice);
const totalProducts = useAppSelector(selectTotalProducts);
```